dbinit.sql

```diff
@@ -0,0 +1,9 @@
+CREATE TABLE "Customer" (
+    id UUID PRIMARY KEY
```

Comment:
I was thinking it would be more common to generate IDs using `DEFAULT gen_random_uuid()`. But then that would be more annoying to insert. @otan what do you think?